src/events/module/subCriteriaModule.ts

```diff
@@ -1,30 +1,32 @@
-import { Module } from "@nestjs/common";
-import { SubCriteriaController } from "../controller/subCriteriaController";
-import { SubCriteriaService } from "../service/subCriteriaService";
-import { TypeOrmModule } from "@nestjs/typeorm";
-import { ConfigModule } from "@nestjs/config";
-import { SubCriteria } from "../entity/subCriteria";
-import { SubCriteriaRepository } from "../repository/subCriteriaRepository";
+import {Module} from '@nestjs/common';
+import {SubCriteriaController} from '../controller/subCriteriaController';
+import {SubCriteriaService} from '../service/subCriteriaService';
+import {TypeOrmModule} from '@nestjs/typeorm';
+import {ConfigModule} from '@nestjs/config';
+import {SubCriteria} from '../entity/subCriteria';
+import {SubCriteriaRepository} from '../repository/subCriteriaRepository';
+import {Pivot} from '../entity/pivot';
 
 @Module({
   imports: [
     ConfigModule.forRoot(),
-    TypeOrmModule.forFeature([ SubCriteria ]),
+    TypeOrmModule.forFeature([SubCriteria, Pivot]),
     TypeOrmModule.forRoot({
-      type: "mysql",
+      type: 'mysql',
       host: process.env.DB_HOST,
       port: +process.env.DB_PORT,
```

Comment:
We can change all places like this to
```suggestion
      port: Number(process.env.DB_PORT),
```